fy22_cecil_b_moore/config.py

```diff
@@ -0,0 +1,42 @@
+from configparser import ConfigParser
+
+
+def config(filename="database.ini", section="postgresql"):
```

Comment:
`config()` and `config2()` look like the same function, just using a different `section` kwarg. Next time write this as one function and then just instantiate it twice, each one with the necessary kwarg. e.g.:
```
def config():
    # ...

config1 = config(section="foo")
config2 = config(section="bar")
```